.changeset/sour-mirrors-chew.md

```diff
@@ -0,0 +1,9 @@
+---
+'@hyperlane-xyz/cli': major
+'@hyperlane-xyz/helloworld': major
+'@hyperlane-xyz/sdk': major
+'@hyperlane-xyz/utils': major
+'@hyperlane-xyz/widgets': major
```

Comment:
only sdk/utils/widgets were touched in https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/5650
```suggestion
'@hyperlane-xyz/sdk': major
'@hyperlane-xyz/utils': major
'@hyperlane-xyz/widgets': major
```